extensions/typescript/package.json

```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "publisher": "vscode",
   "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
+  "enableProposedApi": true,
```

Comment:
I'm still seeing: 

<img width="972" alt="screen shot 2017-05-23 at 6 51 21 pm" src="https://cloud.githubusercontent.com/assets/12821956/26383659/eeca306e-3fe8-11e7-98bc-dfec5aa0d240.png">

Was this a recent change?